### ISSUE TITLE
FFmpegFrameGrabber could not play a rtmp stream 

### ISSUE THREAD

**Issue by felixliu87** [bytedeco/javacv#160]
```java
	 FFmpegFrameGrabber grabber = new FFmpegFrameGrabber("rtmp://localhost/serverapps/instancename/");
    	 grabber.setFormat("mp4");
    
    	 grabber.start();
```
It can connect red5 server succesfully,but founds error below：
```
Exception in thread "main" org.bytedeco.javacv.FrameGrabber$Exception: avformat_open_input() error -1313558101: Could not open input "rtmp://10.11.11.47:1935/VideoMettingServer/felix1/". (Has setFormat() been called?)
	at org.bytedeco.javacv.FFmpegFrameGrabber.startUnsafe(FFmpegFrameGrabber.java:393)
	at org.bytedeco.javacv.FFmpegFrameGrabber.start(FFmpegFrameGrabber.java:341)
	at Demo3.main(Demo3.java:39)
RTMP_ReadPacket, failed to read RTMP packet header
RTMP_ReadPacket, failed to read RTMP packet header
```

**Comment by saudet** [comment]
What does ffplay display if you pass it that URL?

**Comment by felixliu87** [comment]
import java.awt.image.BufferedImage;
import java.io.BufferedInputStream;
import java.io.ByteArrayInputStream;
import java.io.File;
import java.io.InputStream;
import java.nio.ByteBuffer;

import javax.imageio.ImageIO;
import org.apache.log4j.Logger;
import org.bytedeco.javacpp.avcodec;
import org.bytedeco.javacpp.opencv_core.Buffer;
import org.bytedeco.javacpp.opencv_core.IplImage;
import org.bytedeco.javacv.CanvasFrame;
import org.bytedeco.javacv.FFmpegFrameGrabber;
import org.bytedeco.javacv.FrameGrabber;
import org.bytedeco.javacv.Frame;
import org.jcodec.common.model.Picture;


public class Demo3 {
	
	private static final Logger log = Logger.getLogger(Demo3.class);
    public static void main(String[] args) throws Exception {

    
    	FFmpegFrameGrabber grabber = new FFmpegFrameGrabber("rtmp://10.11.11.45/VideoMettingServer/felix1 live=1");

    	grabber.start(); 
    	 System.out.println(grabber.grabFrame().image);
    	   CanvasFrame cFrame = new CanvasFrame("Capture Preview", CanvasFrame.getDefaultGamma() / grabber.getGamma());

           Frame capturedFrame = null;

           // While we are capturing...
           while ((capturedFrame = grabber.grabFrame()) != null)
           {
               if (cFrame.isVisible())
               {
                   // Show our frame in the preview
                   cFrame.showImage(capturedFrame);
               }
           }

           cFrame.dispose();
        
           grabber.stop();
    }
}


Ffplay display is ok,but i  cannot grab frame

Error below：

Exception in thread "main" java.lang.NullPointerException
	at java.awt.image.BufferedImage.<init>(BufferedImage.java:629)
	at org.bytedeco.javacv.Java2DFrameConverter.getBufferedImage(Java2DFrameConverter.java:646)
	at org.bytedeco.javacv.CanvasFrame.showImage(CanvasFrame.java:366)
	at org.bytedeco.javacv.CanvasFrame.showImage(CanvasFrame.java:363)
	at Demo3.main(Demo3.java:43)

**Comment by saudet** [comment]
Could you try with the latest version of `Java2DFrameConverter` here: https://github.com/bytedeco/javacv/commit/918c7540482a87be6b6c83b10abad660b551c140
And let me know how how that goes? It should fix this issue, but I'm not sure. Thanks!

**Comment by saudet** [comment]
The fix above is now in version 1.0, so could you please try again with version 1.0 and let me know if that works or not? Thank you!

**Comment by saudet** [comment]
Given the lack of feedback, I'm guessing everything works fine with JavaCV 1.0. Let me know if this is not the case though, thank you!
